executor/partition_table_test.go

```diff
@@ -949,6 +949,181 @@ func (s *partitionTableSuite) TestSplitRegion(c *C) {
 	tk.MustPartition(`select * from thash where a in (1, 10001, 20001)`, "p1").Sort().Check(result)
 }
 
+func (s *partitionTableSuite) TestDML(c *C) {
+	if israce.RaceEnabled {
```

Comment:
B.T.W, we have so many parallel executors, apply, aggregation, partition table reader ... 
All the data race test is skipped?